app/component/navigation/index-sub-navigation.cjsx

```diff
@@ -1,78 +1,22 @@
-React       = require 'react'
-TimeStore   = require '../../store/time-store' 
-TimeActions = require '../../action/time-action'
+React         = require 'react'
```

Comment:
I see, right. When I opened the link to the comment from chat, Github showed only line 1. "View full changes" helps :-)